### PR TITLE
Align footer on large screens (take 2)

### PR DESCRIPTION
**Pull request by ssandbekkhaug**
### Description

Recreating #8172 since I messed up that PR and accidentally "merged" it into another PR (it was not submitted/merged into the main codebase).

The latest revision copies heavily from [USWDS slim footer](https://designsystem.digital.gov/components/footer/#slim-footer-2), as Sijia suggested. If the USWDS standard and CiviForm [UX mocks](https://www.figma.com/design/RmhVJCLgAKWGENJLWWAtNS/CiviForm%3A-Components%2C-Styles?node-id=1836-7144&m=dev) have a small conflict, use USWDS.

On large screens, put footer links in the center column as shown in the mocks and USWDS site. Alignment follows USWDS as much as possible (e.g. the technical support link is a contact link that overflows 1 line. This is an intentional decision to follow USWDS instead of rewriting the footer to expand the address div.).

On small screens, each link gets its own row.

## Release notes

Styling the footer for North Star, including adaptations for different screen sizes

#### General

Read the full guidelines for PRs [here](https://github.com/civiform/civiform/wiki/Technical-contribution-guide#creating-a-pull-request)

- [x] Added the correct label: < feature | enhancement | bug | under-development | dependencies | infrastructure | ignore-for-release | database >
- [x] Assigned to a specific person, `civiform/developers`, or a [more specific round-robin list](https://github.com/civiform/civiform/wiki/Technical-contribution-guide#adding-reviewers)
- [x] Added an additional reviewer from outside your organization as FYI (if the primary reviewer is in the same organization as you)
- [x] Created unit and/or browser tests which fail without the change (if possible)
- [x] Performed manual testing (Chrome and Firefox if it includes front-end changes)
- [x] Extended the README / documentation, if necessary

#### User visible changes

- [x] Followed steps to [internationalize](https://github.com/civiform/civiform/wiki/Internationalization-%28i18n%29#application-strings) any new strings
  - [x] Added context strings to new [messages](https://github.com/civiform/civiform/blob/main/server/conf/i18n/messages)
  - [x] Didn't use a message in applicant facing code that isn't translated yet (unless behind a flag)
- [x] Wrote browser tests using the [validateAccessibility](https://sourcegraph.com/github.com/civiform/civiform/-/blob/browser-test/src/support/index.ts?L437:14&subtree=true) method
- [x] Tested on mobile view. See [mobile device mode](https://developer.chrome.com/docs/devtools/device-mode/)
- [x] Manually tested at 200% size
- [x] Manually evaluated tab order

### Instructions for manual testing

n/a

### Issue(s) this completes

Contributes to #8155 and #8070

### CODE REVIEW

**Comment by ssandbekkhaug** [comment]
* **#8189** <a href="https://app.graphite.dev/github/pr/civiform/civiform/8189?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @ssandbekkhaug and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: server/app/assets/stylesheets/northstar/_uswds-theme-custom-styles.scss [hunk 1]
```diff
@@ -89,6 +89,15 @@ See the radio-option below as an example.
   @include u-height(1);
 }
 
+.cf-footer-primary-link {
+  @include u-text('base-darkest');
+  @include u-font-size('body', 'sm');
+  font-style: normal;
```
Comment (dkatzz) [inline server/app/assets/stylesheets/northstar/_uswds-theme-custom-styles.scss:100]: **question:** Do we still need these properties with the u-font-size inclusion?

File: server/app/views/applicant/NavigationFragment.html [hunk 2]
```diff
@@ -215,30 +215,41 @@
...
+      <div class="grid-row">
+        <!-- Intentionally NOT using grid-col-8 to allow the technical support link to be positioned on the right -->
+        <div class="tablet:grid-col">
+          <div class="display-flex flex-wrap cf-footer-help-links">
+            <a
+              th:href="${adminLoginUrl}"
+              th:text="#{link.adminLogin}"
+              class="padding-2 cf-footer-primary-link"
+            ></a>
+            <a
```
Comment (dkatzz) [inline server/app/views/applicant/NavigationFragment.html:240]: **question:** I didn't see the "About CiviForm" link in the mocks. Confirming we still want it?

**Review by dkatzz — APPROVED** [review]

File: server/app/views/applicant/NavigationFragment.html (hunk 2 above)
Comment (ssandbekkhaug) [reply server/app/views/applicant/NavigationFragment.html:240]: I asked Sijia, they said to delete it

File: server/app/assets/stylesheets/northstar/_uswds-theme-custom-styles.scss (hunk 1 above)
Comment (ssandbekkhaug) [reply server/app/assets/stylesheets/northstar/_uswds-theme-custom-styles.scss:100]: Realized I could use usa-footer__primary-link class

**Pull request merged**